pkg/exporter/azureblob_exporter.go

```diff
@@ -19,23 +19,60 @@ const (
 )
 
 // AzureBlobExporter defines an Azure Blob Exporter
-type AzureBlobExporter struct{}
+type AzureBlobExporter struct {
+	BaseExporter
```

Comment:
make exporter design consistent with collector and diagnoser as there is no reason for it to be different, required in order to add a constructor